libobs-d3d11/CMakeLists.txt

```diff
@@ -25,9 +25,9 @@ target_sources(
 
 configure_file(cmake/windows/obs-module.rc.in libobs-d3d11.rc)
 target_sources(libobs-d3d11 PRIVATE libobs-d3d11.rc)
-target_compile_options(libobs-d3d11 PRIVATE /wd4595)
```

Comment:
The intention is to downgrade this warning back to an actual warning so that somebody can investigate the severity of the warning and fix the underlying error:

`warning C4595: 'operator new[]': non-member operator new or delete functions may not be declared inline`

Inlining `new` and `delete` is effectively "banned" in C++, so we should fix this instead:

> The program is ill-formed, no diagnostic required if more than one replacement is provided in the program for any of the replaceable allocation function, or if a replacement is declared with the inline specifier.

Reply:
Interesting. Wonder what happened there. Seems to originate from here unless I'm mistaken.
https://github.com/obsproject/obs-studio/commit/db895092ed60a96c64bdf06823ee144867c4e267#diff-31d6cb641dc2ed5bf5e13c32bd469d3b31464d8c80c9fde4f650fd367c849839R28

Reply:
Correct, I originally just silenced it, but in the course of the cleanup I figured out that I can revert the "warnings as error" option as a target option (because MSVC does not have an equivalent to `-Wno-error=<warning>`).

Reply:
Got it.